src/NetworkListener.h

```diff
@@ -1,5 +1,5 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2015-2016. All Rights Reserved.                        */
+/* Copyright (c) FIRST 2017-2016. All Rights Reserved.                        */
```

Comment:
Yeah, that happened because the regex I wrote just matched "FIRST 201x" and replaced it with "FIRST 2017" - it was ignoring the "-2016" after that.